src/vfs2.c

```diff
@@ -1468,7 +1468,6 @@ int vfs2_apply(sqlite3_file *file, struct vfs2_wal_slice stop)
 
 int vfs2_poll(sqlite3_file *file,
 	      dqlite_vfs_frame **frames,
-	      unsigned *n,
```

Comment:
I think we need a comment describing the function that states that `len(frames) == sl.len`.

Reply:
Done, and added comments to vfs2.h where they were previously missing while I was at it.